modules/settings.js

```diff
@@ -13,63 +12,120 @@ function Settings() {
   };
 
   // Load options if present
-  (function() {
-    var retrievedOptions = localStorage.getItem('ffosbrOptions');
+  var retrievedOptions = localStorage.getItem('ffosbrOptions');
 
-    if (retrievedOptions !== null) {
-      self.options = JSON.parse(retrievedOptions);
-    }
-  })();
+  if (retrievedOptions !== null) {
+    this.options = JSON.parse(retrievedOptions);
```

Comment:
Do we need any sort of validation here to make sure what we parsed is a valid options object?